### PR TITLE
manifest-list.md: in the manifest-> in `manifests`

### PR DESCRIPTION
**Pull request by coolljt0725**
The filed name is `manifests` but not manifest.

Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

File: manifest-list.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This section defines the `application/vnd.oci.image.manifest.list.v1+json` [medi
   This REQUIRED property contains a list of manifests for specific platforms.
   While the property MUST be present, the size of the array MAY be zero.
 
-  Each object in the manifest is a [descriptor](descriptor.md) with the following additional properties and restrictions:
+  Each object in the manifests is a [descriptor](descriptor.md) with the following additional properties and restrictions:
```
Comment (wking) [inline manifest-list.md]: How about:

> Each object in `manifests` is…

**Comment by coolljt0725** [comment]
@wking updated

File: manifest-list.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This section defines the `application/vnd.oci.image.manifest.list.v1+json` [medi
   This REQUIRED property contains a list of manifests for specific platforms.
   While the property MUST be present, the size of the array MAY be zero.
 
-  Each object in the manifest is a [descriptor](descriptor.md) with the following additional properties and restrictions:
+  Each object in manifests is a [descriptor](descriptor.md) with the following additional properties and restrictions:
```
Comment (wking) [inline manifest-list.md]: I think we [want backticks around `manifests`][1].

[1]: https://github.com/opencontainers/image-spec/pull/430#discussion_r85949960

Comment (coolljt0725) [inline manifest-list.md]: @wking  oh, sorry, miss that, will do.

**Comment by coolljt0725** [comment]
@wking  done

**Comment by wking** [comment]
6411c007 looks good to me, although the commit message is slightly stale.

**Comment by coolljt0725** [comment]
@wking  commit message has updated

**Comment by wking** [comment]
586f0824 looks good to me.

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
LGTM

**Pull request merged**